packages/picasso/src/PageHamburger/PageHamburgerContext.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode, createContext, useContext, useState } from 'react'
+
+export interface HamburgerContextProps {
+  hamburgerId: string
+  showHamburger?: boolean
+  setShowHamburger?: (showHamburger: boolean) => void
+}
+
+const PageHamburgerContext = createContext<HamburgerContextProps>({
+  hamburgerId: 'hamburger',
+})
+
+interface Props {
+  children: ReactNode
+  hamburgerId: string
+}
+
+export const PageHamburgerContextProvider = ({
+  children,
+  hamburgerId,
+}: Props) => {
+  const [showHamburger, setShowHamburger] = useState<boolean>(false)
+
+  const context = {
+    hamburgerId,
+    showHamburger,
+    setShowHamburger,
+  }
```

Comment:
```suggestion
  const [isHamburgerVisible, setHamburgerVisiblity] = useState<boolean>(false)
  
  const showHamburger = () => setHamburgerVisiblity(true)

  const context = {
    hamburgerId,
    showHamburger,
    isHamburgerVisible,
  }
```

WDYT? `showHamburger` evocates a function in me 🤔

Reply:
I changed the naming, also added the hook `usePortalToHamburger` please check it out 🙇